src/pods/import-collection/edit-table.business.ts

```diff
@@ -0,0 +1,109 @@
+import { GenerateGUID } from '@/core/model';
+
+export type FieldType =
+	| 'any'
+	| 'array'
+	| 'binData'
+	| 'bool'
+	| 'date'
+	| 'dbPointer'
+	| 'decimal'
+	| 'double'
+	| 'enum'
+	| 'int'
+	| 'javascript'
+	| 'long'
+	| 'maxKey'
+	| 'minKey'
+	| 'null'
+	| 'object'
+	| 'objectId'
+	| 'regex'
+	| 'string'
+	| 'symbol'
+	| 'timestamp'
+	| 'undefined';
+
+export interface FieldVm {
+	id: string;
+	PK: boolean;
+	FK: boolean;
+	name: string;
+	type: FieldType;
+	children?: FieldVm[];
+	isCollapsed?: boolean;
+	isArray?: boolean;
+	isNN?: boolean;
+}
```

Comment:
Let's move FieldType and FieldVm to the file

edit-table.model.ts

By using this we simplify this file and segregate